src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class User {
+
+  private final String fname;
```

Comment:
Please use more descriptive variable named (eg firstName) throughout.

Reply:
Done

Reply:
It seems that there is a missing commit where these changes have been done that has not yet been pushed to the server.

Reply:
Yes, I just have a few a problems with my cloud project right now. I hope I'll fix them soon and then add the commit.